src/pages/ProposalDetails/ProposalDetails.component.tsx

```diff
@@ -72,53 +72,55 @@ export const ProposalDetailsComponent = ({
         </Box>
       }
     >
-      <Grid container>
-        <Grid item sm={12} sx={{ p: ({ spacing }) => spacing(0, 2) }}>
-          <VotesRatioBlock />
-        </Grid>
+      <Box sx={{ p: ({ spacing }) => spacing(0, 2), mb: 1 }}>
+        <VotesRatioBlock />
+      </Box>
+
+      <Grid
+        container
+        display="grid"
+        gridTemplateColumns="repeat(2, 1fr)"
+        gap={2}
+      >
+        <ForVotesContainer />
+        <AgainstVotesContainer />
+
+        <Container sx={{ p: 2, minHeight: 300 }}>
+          <Typography variant="body2" sx={{ mb: 2, minHeight: 50 }}>
+            {proposal.description}
```

Comment:
For the future, we can add a collapse component here with some default height - connected to the above Container height

Reply:
That's not a bad idea actually

> and do want to keep the mobile/tablet screen resizable? ![Selection_096](https://user-images.githubusercontent.com/100695408/162403535-412d593a-5503-493c-bdfd-7a83dd7949a1.png)

We're not supporting mobiles for now

Reply:
> For the future, we can add a collapse component here with some default height - connected to the above Container height

That's a really good idea. Will add it

Reply:
> We're not supporting mobiles for now

I mean, before changes columns were under each other on mobile but ok

Reply:
> > We're not supporting mobiles for now
> 
> I mean, before changes columns were under each other on mobile but ok


I restored the flex version since height of the blocks can be fixed now because of the scroll